src/store/keyboard/context.js

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext, useState } from 'react';
+
+const KeyboardContext = createContext();
+
+export const KeyboardProvider = ({ children }) => {
+  const [isOptionKey, setIsOptionKey] = useState(false);
+
+  const initialvalue = {
+    isOptionKey,
+    setIsOptionKey,
```

Comment:
Do we need to put this in context?

Reply:
I do think that we have to put it there, because we need to change it there, correct me if I am understanding it wrong

Reply:
> because we need to change it there

Where?

Reply:
in the div that we are providing below in the keyboard context

Reply:
I think it'll work. Can you run and verify please?